docs/rdiff-backup.1

```diff
@@ -529,8 +536,13 @@ Check all the data in the repository at the given time by computing
 the SHA1 hash of all the regular files and comparing them with the
 hashes stored in the metadata file.
 .TP
-.B "-V, \-\-version"
-Print the current version and exit
+.B "-V, \-\-version, \-\-versions"
+Print the current version and exit.
+.B \-\-versions
```

Comment:
I'm not a big fan of this --versions option. Could we stick with `--version` and use `--verbose` to display more details ? Or something similar.

Reply:
Version is output before the logging mechanism is setup, so difficult to do it like this. I also wanted to make sure that we get all information we need without having to explain too many cases. I could use solely the API version to decide about the output given... Again, my only driver was to be backward compatible. Let me think about it.

Reply:
I removed --versions, `--version` outputs the same thing as long as the actual `api-version` is above 200.